### ISSUE TITLE
[Request] Everybody's Golf/Hot Shot Golf 

### ISSUE THREAD

**Issue by Instandhaltung** [Electry/VitaGrafix#14]
I would die for a resolution improvement for this game. Could you look into it? :) Thanks!

**Comment by Instandhaltung** [comment]
Aaw, thanks for trying! Maybe you'll still find a way :-*

**Comment by EnekoT2001** [comment]
Any updates? Now that VG is able to change MSAA it's possible? I found that it uses 4MSAA and a resolution of 640x368.

Here I have screenshots of VGi just in case it helps: https://imgur.com/a/cdc9zcj
The vita has 9MB of RAM and 5MB of VRAM free in gameplay, I think it's possible to work on. Right?

**Comment by Electry** [comment]
I got it working at 960x544 (with terribly broken UI), but had to disable msaa. Even with msaa off it only ran @ 20-30 fps at max clocks.

There is just not enough memory for full 4x msaa + 544p.

**Comment by EnekoT2001** [comment]
> I got it working at 960x544 (with terribly broken UI), but had to disable msaa. Even with msaa off it only ran @ 20-30 fps at max clocks.
> 
> There is just not enough memory for full 4x msaa + 544p.

Oh okay, with full clocks you mean lolicon at 500mhz?
And maybe is like NFS: MW and works better with 4x MSAA at 832x480? If you try to disable MSAA at any res the UI brokes? Thank you

**Comment by adrianbass360** [comment]
> I got it working at 960x544 (with terribly broken UI), but had to disable msaa. Even with msaa off it only ran @ 20-30 fps at max clocks.
> 
> There is just not enough memory for full 4x msaa + 544p.

Can you fix it to run at 720x544 or 840x544? That would be amazing, thanks!

**Comment by adrianbass360** [comment]
> I got it working at 960x544 (with terribly broken UI), but had to disable msaa. Even with msaa off it only ran @ 20-30 fps at max clocks.
> 
> There is just not enough memory for full 4x msaa + 544p.

Hey Electry, if you're still alive, mind sharing where you left off? =D

**Comment by Bio1998** [comment]
Any news? I like this game but It feels the need of better resolution.

**Comment by Girotin** [comment]
I think he stopped working on this project, it will finish one year now that literally nothing happens to this repo.

**Comment by Sal-gamer** [comment]
> Acho que ele parou de trabalhar nesse projeto, vai terminar um ano agora que literalmente nada acontece com esse repo.

I think he did not stop the project, from 2018 to 2019 he did the same thing for almost a year, many people thought he had given up even I was scared, then he comes back with everything, he lets go  the patch from unchartd, mortal Kombat, Injustice and other top

**Comment by adrianbass360** [comment]
@Electry ELECTRY, we pretty much won't be having any new PS Vita game releases anymore lol. If you don't mind, would you help us one last time? Minecraft and Hot Shots Gold could really use an upscale. I'm sure Hot Shots could run great @ 720x544 with 4xMSAA, similar to NFS:MW.

**Comment by Promesty** [comment]
> I think he stopped working on this project, it will finish one year now that literally nothing happens to this repo. 

You are right, probably Asian people still working on how to modify the codes and stuff, if looking for some help to figure how to do it and try alternative resolutions to different games, do you want to help me, we can create something like a Discord group chat and try!
